virt/vm_networking/virt-connecting-vm-to-primary-udn.adoc

```diff
@@ -0,0 +1,44 @@
+:_mod-docs-content-type: ASSEMBLY                               
+[id="virt-connecting-vm-to-primary-udn"]                                    
+= Connecting a virtual machine to a primary user-defined network                                                
+include::_attributes/common-attributes.adoc[]                   
+:context: virt-connecting-vm-to-primary-udn                                    
+                                                                
+toc::[]                
+
+You can connect a virtual machine (VM) to a user-defined network (UDN) on the VM's primary interface. The primary user-defined network replaces the default pod network in your specified namespace. Unlike the pod network, you can define the primary UDN per project, where each project can use its specific subnet and topology.
+
+{VirtProductName} supports the namespace-scoped `UserDefinedNetwork` and the cluster-scoped `ClusterUserDefinedNetwork` custom resource definitions (CRD). 
+
+Tenant owners can configure a primary `UserDefinedNetwork` CRD to create a network that isolates their namespace from other namespaces without requiring network policies. Cluster administrators can use the `ClusterUserDefinedNetwork` CRD to create a shared OVN network across multiple namespaces.
+
+[NOTE]
+====
+You must add the `k8s.ovn.org/primary-user-defined-network` label when you create a namespace that is to be used with user-defined networks.
```

Comment:
@maiqueb This is new to me.
Until now, we only needed to create a UDN in a certain namespace, in order to make this namespace a UDN one.
Do you know anything else?

Reply:
Replied below.